src/ui_fsmenu/options.cc

```diff
@@ -426,6 +452,14 @@ void FullscreenMenuOptions::layout() {
 		game_clock_.set_desired_size(tab_panel_width, game_clock_.get_h());
 		numpad_diagonalscrolling_.set_desired_size(
 		   tab_panel_width, numpad_diagonalscrolling_.get_h());
+
+		showhide_info_.set_desired_size(tab_panel_width, show_buildhelp_.get_h());
```

Comment:
Can you try adding these and all other similar elements with a different resize option to the box (kFullsize of kFillspace, I can never remember which is which)? We should then be able to get rid of all these set_desired_size calls.

Reply:
It does not work for all elements, but for checkboxes, which we mostly have, it does.